src/atoms/Thumbnail/Thumbnail.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ThumbnailAtom from './Thumbnail';
+
+describe('Thumbnail Atom works properly', () => {
+  it('renders the correct image', () => {
+    render(<ThumbnailAtom thumbnail="someURL" />);
+    expect(screen.getByTestId('thumbnail')).toHaveStyle('background-image: url(someURL)');
+  });
```

Comment:
I'd add a test for checking the height. If your styled-component changes in the future, you can catch that change with the height test. This way, developers are aware that they might break the Thumbnail height.